packages/deploy-config-generator-shared/src/prompts/index.ts

```diff
@@ -18,8 +18,8 @@ export function getConfirmConfigUpdatePrompt(configType?: string): Question[] {
     return [
```

Comment:
Im not sure these prompts should be here as they are extension generator specific?

Reply:
Going to resolve conversation and update internal ticket with comments.